config/README.md

```diff
@@ -0,0 +1,119 @@
+# Dependencies installation
+
+For a faster creation of the environment (and Conda environments in general),
+you can also install [Mamba][mamba] on top of Conda. In that case, replace
+`conda` with `mamba` in the commands below (particularly in 
+`conda env create`).
```

Comment:
It's a bit of a weird way to start instructions. I would rather but this as a comment below (and then refer to the "commands above").